src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
```

Comment:
The function `convertToObject` is missing a description. It's a good practice to add a comment describing the function's purpose, parameters, and return value.